JBIDE-22291 Deploy Docker Wizard: Add connection info

scabanovich opened the pull request.

adietish commented on plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImagePage.java:
```diff
@@ -268,6 +283,32 @@ protected IStatus validate() {
+		final IObservableValue connnectionTextObservable = WidgetProperties.text(SWT.None).observe(connectionText);
+		final IObservableValue connnectionObservable = BeanProperties.value(IDeployImagePageModel.PROPERTY_CONNECTION).observe(model);
+		ValueBindingBuilder.bind(connnectionTextObservable)
+			.to(connnectionObservable)
```
I'd add .notConvertingParticipant() here so that no update strategy text -> model is being attached.

scabanovich replied:
Ok, done.

scabanovich commented:
By advice of @fbricon , added info docker connection for the case when wizard is started from Docker explorer, and moved docker connection widgets after separator to image and resource name.

fbricon commented:
Merged as https://github.com/jbosstools/jbosstools-openshift/commit/a168690085d3a218bd149c54697bdafb3fc63546, after removing the readonly text's borders
